### PR TITLE
Fix alignment and whitespace issues

### PR DESCRIPTION
**Pull request by bdunne**
Running the tests prior to the changes results in:

image

### CODE REVIEW

File: spec/fstab_spec.rb [hunk 1]
```diff
@@ -63,4 +63,50 @@
       subject.instance.write!
     end
   end
+
+  describe "integration test" do
```
Comment (carbonin) [inline spec/fstab_spec.rb]: This is still just testing `#write` though ... right?

File: spec/fstab_spec.rb [hunk 2]
```diff
@@ -46,20 +46,84 @@
...
+                /dev/mapper/VG--MIQ-lv_os                     /  xfs defaults 0 0
+UUID=02bf07b5-2404-4779-b93c-d8eb7f2eedea                 /boot  xfs defaults 0 0
+              /dev/mapper/VG--MIQ-lv_home                 /home  xfs defaults 0 0
+               /dev/mapper/VG--MIQ-lv_tmp                  /tmp  xfs defaults 0 0
+               /dev/mapper/VG--MIQ-lv_var                  /var  xfs defaults 0 0
+           /dev/mapper/VG--MIQ-lv_var_log              /var/log  xfs defaults 0 0
+     /dev/mapper/VG--MIQ-lv_var_log_audit        /var/log/audit  xfs defaults 0 0
+               /dev/mapper/VG--MIQ-lv_log /var/www/miq/vmdb/log  xfs defaults 0 0
+              /dev/mapper/VG--MIQ-lv_swap                  swap swap defaults 0 0
+EOC
```
Comment (carbonin) [inline spec/fstab_spec.rb]: Can you fix all of these to use `<<~` since we only support ruby >2.3 now?

Comment (carbonin) [reply spec/fstab_spec.rb]: This should also fix a bunch of the rubocop issues I think.

**Comment by carbonin** [comment]
Ah, also could you address the rubocop comments for the newly added code?

File: spec/fstab_spec.rb (hunk 1 above)
Comment (bdunne) [reply spec/fstab_spec.rb]: Not really, it is also testing the parsing, max column length logic, etc.

File: lib/linux_admin/fstab.rb [hunk 3]
```diff
@@ -44,30 +44,31 @@ def columns
...
-        with_index { |col, i| col.to_s.rjust(max_lengths[i]) }.join(" ")
+        with_index { |col, i| col.to_s.rjust(max_lengths[i]) }.join(" ").rstrip
     end
   end
 
   class FSTab
     include Singleton
 
-    attr_accessor :entries
-    attr_accessor :maximum_column_lengths
```
Comment (carbonin) [inline lib/linux_admin/fstab.rb]: Maybe we should keep a method for this so we don't change API compatibility.

Comment (bdunne) [reply lib/linux_admin/fstab.rb]: Done

File: lib/linux_admin/fstab.rb [hunk 4]
```diff
@@ -44,30 +44,35 @@ def columns
...
       refresh
     end
 
+    def entries
+      @entries ||= LinuxAdmin::FSTab::EntryCollection.new
+    end
+
+    def maximum_column_lengths
+      entries.maximum_column_lengths
+    end
```
Comment (jrafanie) [inline lib/linux_admin/fstab.rb]: Do we care about the setter methods that we're losing by implementing only the getter method?

Comment (carbonin) [reply lib/linux_admin/fstab.rb]: I don't even know what the setter would do in this case ... Nothing productive for sure.

Comment (bdunne) [reply lib/linux_admin/fstab.rb]: I don't think the setter should have ever been public api

**Comment by carbonin** [comment]
@bdunne do you want to take care of the release?

**Comment by simaishi** [comment]
@bdunne Travis is failing in manageiq-schema. Looks like this PR is the cause?

```
  1) DropMiqServerRhnMirror#up cleans up external files on an appliance
     Failure/Error: described_class.migrate(migration_direction)
       #<File (class)> received :write with unexpected arguments
         expected: ("/etc/fstab", "/dev/sda     / xfs defaults        0        0 \n")
              got: ("/etc/fstab", "/dev/sda     / xfs defaults 0 0\n")
```

https://travis-ci.org/ManageIQ/manageiq-schema/jobs/556038000

**Pull request merged**